.github/workflows/integration-test-object-suite.yaml

```
@@ -39,7 +39,7 @@ jobs:
         uses: ./.github/workflows/tmate_debug
         with:
           use-tmate: ${{ secrets.USE_TMATE }}
-          debug-ci: ${{ contains(github.event.pull_request.labels.*.name, 'debug-ci') }}
+          debug-ci: true
```

If this isn't working for testing, you might try forcing this to be a string type

```suggestion
          debug-ci: 'true'
```